config not working and producing odd error...

using svelte-image in the rollup config like so:

```js

const extensions = ['.svelte', '.md'];

const preprocess = [
  ...image(),
  mdsvex({
    extension: '.md',
    layout: {
      music: 'src/routes/music/_piece.svelte',
      words: 'src/routes/words/_article.svelte',
      tool: 'src/routes/tools/_tool.svelte',
    },
    remarkPlugins: [
      [attr, { scope: 'every' }]
    ]
  })
]
```

...with mdsvex, then passing preprocess like this:

```js
svelte({
  extensions,
  preprocess,
  dev,
  hydratable: true,
  emitCss: true
}),
```

...and getting this error in the terminal when running `npm run dev`:

```zsh
> rykr-svelte@0.0.1 dev
> sapper dev --ext '.svelte .md'

/Users/rchrdnsh/Code/Svelte/RYKR/rollup.config.js:36
        ...image__default['default'](),
                                    ^

TypeError: image__default.default is not a function or its return value is not iterable
    at Object.<anonymous> (/Users/rchrdnsh/Code/Svelte/RYKR/rollup.config.js:36:30)
    at Module._compile (node:internal/modules/cjs/loader:1083:30)
    at Object.require.extensions..js (/Users/rchrdnsh/Code/Svelte/RYKR/node_modules/sapper/dist/create_manifest_data.js:4190:28)
    at Module.load (node:internal/modules/cjs/loader:948:32)
    at Function.Module._load (node:internal/modules/cjs/loader:789:14)
    at Module.require (node:internal/modules/cjs/loader:972:19)
    at require (node:internal/modules/cjs/helpers:88:18)
    at Function.<anonymous> (/Users/rchrdnsh/Code/Svelte/RYKR/node_modules/sapper/dist/create_manifest_data.js:4196:28)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/rchrdnsh/Code/Svelte/RYKR/node_modules/sapper/dist/_commonjsHelpers.js:27:58)
npm ERR! code 1
npm ERR! path /Users/rchrdnsh/Code/Svelte/RYKR
npm ERR! command failed
npm ERR! command sh -c sapper dev --ext '.svelte .md'

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/rchrdnsh/.npm/_logs/2020-12-06T04_31_33_670Z-debug.log
```

...which is super odd and out of my league to understand...


pleeeeeez halp?

XD

--- a/README.md
+++ b/README.md
@@ -27,9 +27,7 @@ import image from "svelte-image";
 
 
 svelte({
-  preprocess: {
-    ...image(),
-  }
+  preprocess: image(),
 })
 ```
 
@@ -79,9 +77,7 @@ export default {
     ...
     svelte({
       ...
-      preprocess: {
-        ...image({...})
-      }
+      preprocess: image({...}),
     }),
     copy({
       targets: [{ src: 'static/g', dest: 'public' }],
@@ -244,12 +240,10 @@ import image from "svelte-image";
 
 
 svelte({
-  preprocess: {
-    ...image({
-      sizes: [200, 400],
-      processFolders: ['people/images']
-    }),
-  }
+  preprocess: image({
+    sizes: [200, 400],
+    processFolders: ['people/images']
+  }),
 })
 ```
 

